app/scripts/models/auth_brokers/fx-desktop-v2.js

```diff
@@ -29,9 +29,6 @@ define(function (require, exports, module) {
     }),
 
     defaultCapabilities: _.extend({}, proto.defaultCapabilities, {
-      browserTransitionsAfterEmailVerification: true,
```

Comment:
`browserTransitionsAfterEmailVerification` is set to true in base.js, no need to duplicate here.